docs/conf/portland/2025/news/welcome-portland-2025.md

```diff
@@ -0,0 +1,112 @@
+---
+template: {{year}}/generic.html
+og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+---
+
+```{eval-rst}
+.. post:: Apr 28, 2025
+   :tags: {{shortcode}}-{{year}}
+```
+
+# Welcome to Write the Docs Portland 2025!
+
+We have officially arrived at conference week! We're excited to gather in Portland for our 13th year and our 3rd time at Revolution Hall. We hope you are as excited as we are! Below is an overview of the conference, as well as a couple of announcements.
+
+## Important Links
+
+The website is full of wonderful information about the conference, venue, and Portland. We encourage you to check out a few of our pages below.
+
+- [Attendee Guide](https://www.writethedocs.org/conf/portland/2025/attendee-guide/)
+- [Visiting Portland](https://www.writethedocs.org/conf/portland/2025/visiting/)
+- [Schedule](https://www.writethedocs.org/conf/portland/2025/schedule/)
+- [Code of Conduct](https://www.writethedocs.org/conf/portland/2025/code-of-conduct/)
+
+```{figure} /_static/conf/images/pics/portland-unconference.jpg
+```
+
+## How to Participate in the Conference
+
+- **Hike:** Join us on the hike on Saturday for our unofficial kick off to the conference. Hike is free, but space is limited. [Register with a ticket](https://ti.to/writethedocs/write-the-docs-portland-2025).
+- **Writing Day:** We already have five [Writing Day Projects](https://www.writethedocs.org/conf/portland/2025/writing-day/#project-list) that have signed up. Drop in or stay for the entire day.
+- **Welcome Reception:** Do not miss our Sunday evening reception! Pick up your badge and enjoy drinks and snacks on us.
+- **Speaker Talks:** Take a look at our schedule and select the talks you want to attend. [View our lineup here](https://www.writethedocs.org/conf/portland/2025/schedule/#monday-may-5). 
+- **Unconference:** Lead or attend a session so you can connect with like-minded folks about topics you care about. We're accepting Monday session sign-ups now. [Learn more about signing up](#monday-unconference-sessions-open).
+- **Lightning Talks:** Do you have an idea, concept, or topic you'd like to share with our community in five minutes? We're now accepting Monday submissions. [Learn more about submitting a talk](#monday-lightning-talk-submissions-open).
+- **Monday Night Social:** Our only offsite gathering is an informal way to relax with fellow attendees, and again enjoy some drinks and snacks on us! Held at the Jupiter NEXT, 9000 E Burnside St.
```

Comment:
Would be good to note the sponsors, of the lightning talks, and social event.

Reply:
Are these separate code blocks?